controllers/argocd/keycloak.go

```diff
@@ -279,6 +279,14 @@ func getKeycloakDeploymentConfigTemplate(cr *argoprojv1a1.ArgoCD) *appsv1.Deploy
 			},
 		},
 	}
+
+	if cr.Spec.NodePlacement != nil {
+		dc.Spec.Template.Spec.NodeSelector = cr.Spec.NodePlacement.NodeSelector
+		dc.Spec.Template.Spec.Tolerations = cr.Spec.NodePlacement.Tolerations
+	}
+
```

Comment:
If `cr.SpecNodePlacement` is nil, should we reset the dc.Spec?

Reply:
That will be take care by the below section of code.